ci/py3.9.yml

```diff
@@ -0,0 +1,35 @@
+name: pangeo-forge
+channels:
+  - conda-forge
+dependencies:
+  - python=3.9
+  - black
+  - boto3
+  - cfgrib
+  - codecov
+  - dask
+#  - distributed
+  - fsspec
+  - h5netcdf
+  - h5py
+  - hdf5
+  - lxml    # Optional dep of pydap
+  - netcdf4
+  - numpy
+  - pandas
+  - pip
+  - prefect
+  - pydap
+  - pynio
```

Comment:
As far I can tell, the latest build of pynio on conda-forge supports `Python <=3.8`. Remove it from the environment fixes the issue (tested on a linux system). 

```suggestion
```